### DISCUSSION TITLE
Where is the Dowloads?

### DISCUSSION THREAD

**Discussion by Ratteler** [aclap-dev/video-downloadhelper#954] [Help and Q&A]
### Your question

This used to work just fine. Now I can't find downloads in any folders they used to show up in. Showing the file folder doesn't work, nor does playing.
Can I get the old WORKING version back? 

### Addon & CoApp version and details

This used to work just fine. Now I can't find downloads in any folders they used to show up in. Showing the file folder doesn't work, nor does playing.
Can I get the old WORKING version back?

**Comment by jcfrog** [comment]
hi,
this should help to find your default directory: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#change-the-default-download-directory

**Reply by GrampaWildWilly** [reply]
You will also want to read #744.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH (@jcfrog Jérôme does).  I am just another user.

The old version is not a working version.  That is why there is a new version.  The new version corrects a lot of bugs.  You don't want the old version.  You just need to get familiar with the new ways of doing things.  The new version is a working version.  We just need to be patient until you discover this fact.
